src/metabase/lib/segment.cljc

```diff
@@ -32,6 +33,14 @@
 (defn- fallback-display-name []
   (i18n/tru "[Unknown Segment]"))
 
+(defmethod lib.metadata.calculation/metadata-method :metadata/segment
+  [query stage-number segment-metadata]
+  (let [base  (binding [lib.metadata.ident/*enforce-idents-present* false]
+                ((get-method lib.metadata.calculation/metadata-method :default) query stage-number segment-metadata))
+        ident (some-> segment-metadata :definition :aggregation-idents (get 0))]
```

Comment:
I remember segments being just filters w/o aggregation. Under what circumstances will `ident` be truthy?